### PR TITLE
Fixes #26168,#25276 - Create ovirt interface/volume if not empty

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: app/models/compute_resources/foreman/model/ovirt.rb [hunk 1]
```diff
@@ -278,8 +278,8 @@ def create_vm(args = {})
       vm = super({ :first_boot_dev => 'network', :quota => ovirt_quota }.merge(args))
 
       begin
-        create_interfaces(vm, args[:interfaces_attributes])
-        create_volumes(vm, args[:volumes_attributes])
+        create_interfaces(vm, args[:interfaces_attributes]) unless args[:interfaces_attributes].empty?
```
Comment (ezr-ondrej) [inline app/models/compute_resources/foreman/model/ovirt.rb:282]: Wouldn't the check be better to put at the beggining of the `create_interface` method?

Comment (shiramax) [reply app/models/compute_resources/foreman/model/ovirt.rb:282]: @ezr-ondrej , I don't mind, but can you explain why it would be better?

**Comment by shiramax** [comment]
[ test foreman ]

**Comment by shiramax** [comment]
[ test foreman ]

File: app/models/compute_resources/foreman/model/ovirt.rb (hunk 1 above)
Comment (ezr-ondrej) [reply app/models/compute_resources/foreman/model/ovirt.rb:282]: In case the method gets called from a different place :) but it's not a nack.

**Review by ezr-ondrej — APPROVED** [review]
Failure unrelated. Merging, thanks @shiramax.

**Pull request merged**